packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -1501,6 +1501,25 @@ export abstract class AbstractMesh extends TransformNode implements IDisposable,
         this._updateBoundingInfo();
     }
 
+    /**
+     * @internal
+     */
+    public _refreshBoundingInfoDirect(extend: { minimum: Vector3; maximum: Vector3 }): void {

```

Comment:
I think it is "extent", not "extend" (?)

Reply:
Even reading the definition I am still unsure lol 
![image](https://github.com/BabylonJS/Babylon.js/assets/16655165/af0aa590-a889-4796-a67b-9a5141292f5a)